target/linux/lantiq/files/arch/mips/boot/dts/lantiq/ar10.dtsi

```diff
@@ -0,0 +1,410 @@
+
```

Comment:
This file should also have spdx license identifier.

Reply:
Would this be ok?
`// SPDX-License-Identifier: GPL-2.0-or-later`